specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/examples/virtualMachineScaleSetExamples/VirtualMachineScaleSet_Get_WithDiskControllerType.json

```diff
@@ -0,0 +1,107 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "vmScaleSetName": "myVirtualMachineScaleSet",
+    "api-version": "2022-08-01"
```

Comment:
$expand has still not been added.

Reply:
Its present in the files changed. 

![image](https://user-images.githubusercontent.com/85781974/185768695-f211c243-06d9-45ef-b247-f1aa39e078e3.png)